drivers/i3c/i3c_shell.c

```diff
@@ -2094,7 +2099,8 @@ static int cmd_i3c_ibi_tir(const struct shell *sh, size_t argc, char **argv)
 
 	return 0;
 }
-
+#endif /* CONFIG_I3C_TARGET */
```

Comment:
is this comment correct?  I thought line 2066 was the end of this section.  Now that I look at it more, maybe 2066 isn't the right comment to the close?

Reply:
yep, good catch, 2066 had the wrong comment  on it